packages/zosjobs/__tests__/__system__/DownloadJobs.system.test.ts

```diff
@@ -84,51 +84,54 @@ describe("Download Jobs - System tests", () => {
     describe("Positive tests", () => {
 
         it("should be able to download a single DD from job output to specified directory", async () => {
-            const downloadDir = outputDirectory + "/downloadsingle/";
             await DownloadJobs.downloadSpoolContentCommon(REAL_SESSION, {
-                outDir: downloadDir,
+                outDir: outputDirectory,
                 jobFile: jesJCLJobFile
             });
-            expect(IO.existsSync(downloadDir)).toEqual(true);
-            const expectedFile = DownloadJobs.getSpoolDownloadFile(jesJCLJobFile, false, downloadDir);
-            expect(IO.existsSync(expectedFile)).toEqual(true);
-            expect(IO.readFileSync(expectedFile).toString()).toContain("EXEC PGM=IEFBR14");
-        });
-
-        it("should be able to download a single DD from job output", async () => {
```

Comment:
Is there a reason why we removed this test?

Reply:
added back test, thank you for catching this